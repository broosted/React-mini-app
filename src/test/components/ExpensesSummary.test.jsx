import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpensesSummary.jsx';

test('should correctly render ExpensesSummary with 1 expense',() => {
const wrapper = shallow(<ExpenseSummary expenseCount={1} expensesTotal={234}/>)

expect(wrapper).toMatchSnapshot();
})



test('should correctly render ExpensesSummary with multiple expenses',() => {
    const wrapper = shallow(<ExpenseSummary expenseCount={23} expensesTotal={2343232423}/>)
    
    expect(wrapper).toMatchSnapshot();   
})