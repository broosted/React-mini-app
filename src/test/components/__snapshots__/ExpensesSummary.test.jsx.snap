// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.34
  </h1>
</div>
`;

exports[`should correctly render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $23,432,324.23
  </h1>
</div>
`;
